src/OrbitGl/OrbitApp.cpp

```diff
@@ -183,6 +185,8 @@ namespace {
 constexpr const char* kNtdllSoFileName = "ntdll.so";
 constexpr const char* kWineSyscallDispatcherFunctionName = "__wine_syscall_dispatcher";
 constexpr std::string_view kGgpVlkModulePathSubstring = "ggpvlk.so";
+const TimeRange kDefaultTimeRange =
+    TimeRange(std::numeric_limits<uint64_t>::min(), std::numeric_limits<uint64_t>::max());
```

Comment:
This can also be constexpr if we mark the TimeRange constructor as constexpr.

Reply:
The constructor doesn't like the ORBIT_CHECK being in there: 

Variable of non-literal type 'std::filesystem::path' cannot be defined in a constexpr constructor before C++2bclang(constexpr_local_var_non_literal_type)
fs_path.h(292, 9): 'path' is not literal because it is not an aggregate and has no constexpr constructors other than copy or move constructors

Reply:
Ah. Makes sense. Thanks for checking.